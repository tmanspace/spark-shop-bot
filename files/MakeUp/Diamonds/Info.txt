Блеск в глазах и бриллианты под глазами\1500