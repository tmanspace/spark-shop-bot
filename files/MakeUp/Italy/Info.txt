Розовый пресет и вечерний макияж с добавлением ресниц по кнопке\3500