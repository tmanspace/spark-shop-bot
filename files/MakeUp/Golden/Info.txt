Макияж, блестки, тени - прекрасный вариант для "выхода"\2500